examples/sveltekit/src/routes/__layout.svelte

```diff
@@ -2,13 +2,13 @@
 	import { goto } from '$app/navigation';
 	import { session } from '$app/stores';
 	import { supabaseClient } from '$lib/db';
-	import { AuthHelper } from '@supabase/auth-helpers-svelte';
+	import { SuperAuthHelper } from '@supabase/auth-helpers-svelte';
```

Comment:
Should this be `SupaAuthHelper`?

Reply:
I couldn't think of a good name, open to suggestions.

Reply:
I think `AuthHelper` works, or `SupaAuthHelper` to stick with the brand 👍